docs-src/cloud/limits.md

```diff
@@ -63,12 +63,11 @@ The following aspects apply at the Namespace level (per Namespace).
 
 **What is the limit of Actions per second?**
 
-The throughput of a Namespace is measured in Actions per second (APS).
-
-The default limit is 200 APS with the ability to spike to 400 APS.
-
-The APS limit is automatically adjusted based on the actual usage over the trailing 7 days.
-The lowest APS limit that ever gets set is 200 APS.
+Each Namespace has a rate limit, which is measured in Actions per second (APS).
+A Namespace may be throttled when its throughput becomes too high.
+Throttling means limiting the rate at which actions are performed to prevent overloading the system.
+A Namespace's default limit is set at 200 APS and automatically adjusts based on recent usage (over the prior 7 days).
+Your throughput limit will never fall below this default value.
```

Comment:
```suggestion
Your APS limit will never fall below this default value.
```

"APS Limit" is the title used in the UI—suggest using it throughout instead of "throughput limit"